src/spellbot/enums.py

```diff
@@ -88,7 +88,8 @@ def __str__(self) -> str:
     OATHBREAKER = 4
     DUEL_COMMANDER = 2
     CEDH = 4, "cEDH"
-    ARCHENEMY = 4
+    ARCHENEMY = 4,
```

Comment:
No need for the comma here, we can remove that.
```suggestion
    ARCHENEMY = 4
```

Reply:
Ah! Good catch.